lgsm/functions/command_postdetails.sh

```diff
@@ -72,63 +72,70 @@ touch "${tmpfile}" || fn_bad_tmpfile
 # in the output.  Otherwise, strip these out as well.
 if ! grep -q "^steampass[= ]\"\"" "${tmpfile}" ; then
-	sed -i -e 's/steampass[= ]"[^"]*/steampass "--stripped--/' "${tmpfile}"
+  sed -i -e 's/steampass[= ]"[^"]*/steampass "--stripped--/' "${tmpfile}"
 fi
 if ! grep -q "^steamuser[= ]\"anonymous\"" "${tmpfile}" ; then
-	sed -i -e 's/steamuser[= ]"[^"]*/steamuser "--stripped--/' "${tmpfile}"
+  sed -i -e 's/steamuser[= ]"[^"]*/steamuser "--stripped--/' "${tmpfile}"
 fi
 
+
 if [ "${posttarget}" == "http://pastebin.com" ] ; then
-   # grab the return from 'value' from an initial visit to pastebin.
-   csrftoken=$(curl -s "${posttarget}" |
-           sed -n 's/^.*input type="hidden" name="csrf_token_post" value="\(.*\)".*$/\1/p')
-   #
-   # Use the csrftoken to then post the content.
-   #
-   link=$(curl -s "${posttarget}/post.php" -D - -F "submit_hidden=submit_hidden" \
-	       -F "post_key=${csrftoken}" -F "paste_expire_date=${postexpire}" \
-	       -F "paste_name=${gamename} Debug Info" \
-               -F "paste_format=8" -F "paste_private=0" \
-               -F "paste_type=bash" -F "paste_code=<${tmpfile}" |
-	       awk '/^location: / { print $2 }' | sed "s/\n//g")
+  fn_print_dots "Posting details to pastbin.com for ${postexpire}"
+  sleep 1
+  # grab the return from 'value' from an initial visit to pastebin.
+  csrftoken=$(curl -s "${posttarget}" |
+          sed -n 's/^.*input type="hidden" name="csrf_token_post" value="\(.*\)".*$/\1/p')
+  #
+  # Use the csrftoken to then post the content.
+  #
+  link=$(curl -s "${posttarget}/post.php" -D - -F "submit_hidden=submit_hidden" \
+        -F "post_key=${csrftoken}" -F "paste_expire_date=${postexpire}" \
+        -F "paste_name=${gamename} Debug Info" \
+        -F "paste_format=8" -F "paste_private=0" \
+        -F "paste_type=bash" -F "paste_code=<${tmpfile}" |
+        awk '/^location: / { print $2 }' | sed "s/\n//g")
 
    # Output the resulting link.
-   fn_print_warn_nl "Visit (and verify) the content posted at ${posttarget}${link}"
+  fn_print_ok_nl "Posting details to pastbin.com for ${postexpire}"
+  echo "  * url: ${posttarget}/${link}"
 elif [ "${posttarget}" == "http://hastebin.com" ] ; then
-   # hastebin is a bit simpler.  If successful, the returned result
-   # should look like: {"something":"key"}, putting the reference that
-   # we need in "key".  TODO - error handling. -CedarLUG
-   link=$(curl -s -d "$(<${tmpfile})" "${posttarget}/documents" | cut -d\" -f4)
-   fn_print_warn_nl "Visit (and verify) the content posted at ${posttarget}/${link}"
+  fn_print_dots "Posting details to hastbin.com for ${postexpire}"
+  sleep 1
+  # hastebin is a bit simpler.  If successful, the returned result
+  # should look like: {"something":"key"}, putting the reference that
+  # we need in "key".  TODO - error handling. -CedarLUG
+  link=$(curl -s -d "$(<${tmpfile})" "${posttarget}/documents" | cut -d\" -f4)
+  fn_print_ok_nl "Posting details to hastbin.com for ${postexpire}"
```

Comment:
postexpire doesn't apply to hastebin.  hastebin expires in one month by default and isn't able to be changed.  (I can double-check that).